app/views/admin/roles/index.html.erb

```diff
@@ -23,53 +23,52 @@
     {
       text: "Organistions"
     },
-    {
-      text: "Role type"
-    },
     {
       text: "Currently appointed"
     },
     {
       text: "Translations"
     },
     {
-      text: "Edit"
+      text: tag.span(class: "govuk-visually-hidden")
```

Comment:
hey, so we should still have text here so screen readers pick it up.

One thing i noticed with this is that it's two separate columns. Whereas in all other places we've got them as one "Actions" column.

I think we should update this to 

```
      text: tag.span("Actions", class: "govuk-visually-hidden")
```

and then merge the edit and delete columns. Wdyt?

Reply:
yeah ..make sense